Src/BootCamp.Chapter/BalanceParser.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceParser
+    {
+        private const string InValidOutput = "N/A.";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'InValidOutput'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)